ddtrace/internal/datadog/profiling/stack_v2/src/stack_v2.cpp

```diff
@@ -35,7 +30,7 @@ stack_v2_stop(PyObject* self, PyObject* args)
     (void)self;
     (void)args;
     Sampler::get().stop();
-    Py_RETURN_NONE
```

Comment:
I got this wrong in the last PR during a sloppy merge.  Curiously, it didn't break my local tests.  Maybe it did, but the failure didn't over-write my sofiles or something.

Sometimes these macros don't require a semicolon.  I went ahead and checked.  This one has required the user to end the statement since 2003.  Oops.  :)

Reply:
Ah oops did I forget the semicolon in my suggestion? Sorry! 😞

Reply:
Not a problem!  We don't have in-CI tests for this yet, so I've been re-running my test harness locally.  Somehow, the way I did it didn't throw an error.  I'll be adding tests next, so hopefully I can rely on my brain a little less.